webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -62,6 +62,12 @@ interface Props {
   project: Project;
 }
 
+type selectionType = 'add' | 'add-all' | 'remove' | 'remove-all' | 'set';
```

Comment:
**nit:** make the type uppercase `SelectionType` to be consistent

Reply:
fixed